### PR TITLE
minor update for raspbian

### PR DESCRIPTION
**Pull request by coolaj86**

### CODE REVIEW

File: bootstrap/_deb_common.sh [hunk 1]
```diff
@@ -9,7 +9,11 @@
 #     - 6.0.10 "squeeze" (x64)
 #     - 7.8 "wheezy" (x64)
 #     - 8.0 "jessie" (x64)
+#   - Raspbian:
+#     - 7.8 (armhf)
 
+apt-get update
+apt-get install -y lsb-release
```
Comment (kuba) [inline bootstrap/_deb_common.sh]: could you please put `apt-get install ...` as a first line of `newer()` and use `--no-install-recommends`?

Comment (coolaj86) [inline bootstrap/_deb_common.sh]: will do

**Comment by kuba** [comment]
Thanks a lot! LGTM.

Did you actually manage to issue a cert on RPi? :)

**Comment by jdkasten** [comment]
Thanks!

**Comment by coolaj86** [comment]
And yes, I did actually get certs on RPi:

https://coolaj86.com/articles/lets-encrypt-on-raspberry-pi/

**Pull request merged**